params.yaml

```diff
@@ -0,0 +1,33 @@
+preprocess_wellcome_science:
+  text_cols: "Title,Synopsis"
+  meta_cols: "Grant_ID,Title"
+train:
+  tfidf-svm:
+    tfidf:
```

Comment:
As you mentioned, this is a bit redundant with our config files. Another way would be for the `params.yaml` to be minimal, just a path to the newest config file. Which one is the best I don't know...

Reply:
e.g.:

```
train:
   model_version:
   config_file_path:
````

then your train function would just look at parameters and trigger opening the config file. The dvc "diff" will be effectively a diff between model versions, but to actually see the different parameters you would have to go into the config file.

Reply:
this way you lose all the experiments feature where you can see the different params but at the same time we do not use that feature. I am leaning more on the camp that we can created params.yaml from the config so essentially convert one to the other